components/encryption/src/errors.rs

```diff
@@ -17,6 +17,10 @@ pub trait RetryCodedError: Debug + Display + ErrorCodeExt + RetryError + Send +
 pub enum Error {
     #[error("Other error {0}")]
     Other(#[from] Box<dyn error::Error + Sync + Send>),
+    // Only when the parsing record is the last one, and the
+    // length is insufficient or the crc checksum fails.
+    #[error("Recoverable types of parsing file dict error")]
```

Comment:
"Recoverable tail record corruption while parsing file dictionary"? I want to emphasize it is the tail record corruption in the error message.